package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "maker-docker-poa",
+  "version": "0.0.1",
+  "description": "Maker SAI compiler and deployer",
+  "author": "Keydonix <team@keydonix.com>",
+  "repository": "http://github.com/keydonix/maker-docker-poa",
+  "main": "output/index.js",
+  "types": "output/index.d.ts",
+  "module": "output/index.js",
+  "browser": "output/browser.js",
+  "license": "GPL-3.0",
+  "directories": {
+    "src": "source"
+  },
+  "scripts": {
+    "build:source": "npx tsc",
+    "build:contracts": "npm run fetch:sai && node output/deployment/compileContracts.js",
+    "build:interfaces": "npm run fetch:sai && node output/tools/generateContractInterfaces.js",
+    "build": "npm run build:source && npm run build:contracts && npm run build:interfaces",
+    "clean": "rimraf output",
```

Comment:
Does this need to be `npx rimraf`?  Can you directly call npm commands from an npm run script without the `npx` prefix?  If so, can we just do `tsc` above for `build:source` rather than `npx tsc`?